src/components/Heading/Heading.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import {headingDriverFactory} from './Heading.driver';
+import {Heading} from './';
+import {Appearance} from './Heading';
```

Comment:
I think appearance should be exported from index as well

Reply:
I decided to do that because `index.ts` is the file we expose for the users. At the moment it doesn't expose the Appearance, only the component itself.

Do you think this should be exposed from there as well?